src/dotnet-install.ps1

```diff
@@ -837,17 +846,19 @@ function Get-AkaMSDownloadLink([string]$Channel, [string]$Quality, [string]$Prod
     Say-Verbose "Retrieving primary payload URL from aka.ms link for channel: '$Channel', quality: '$Quality' product: '$Product', os: 'win', architecture: '$Architecture'." 
    
     #construct aka.ms link
-    $akaMsLink = "https://aka.ms/dotnet/$Channel" 
+    $akaMsLink = "https://aka.ms/dotnet"
+    if ($Internal) {
+        $akaMsLink += "/internal"
+    }
+    $akaMsLink += "/$Channel"
     if (-not [string]::IsNullOrEmpty($Quality)) {
         $akaMsLink +="/$Quality"
     }
-
     $akaMsLink +="/$Product-win-$Architecture.zip"
     Say-Verbose  "Constructed aka.ms link: '$akaMsLink'."
 
     #get HTTP response
     $Response= GetHTTPResponse -Uri $akaMsLink -HeaderOnly $true -DisableRedirect $true
-    Say-Verbose "Received response:`n$Response"
```

Comment:
Response has credentials in the redirect link, we do not want to log it.

Reply:
Would it be useful to know that the response was received? If yes, maybe we can just log "Response received." without printing the content.

Reply:
Yes, but this line actually will be back when I remove credentials from aka.ms link, it would be safe then.